### PR TITLE
Create NIBM/2022-current.md

### PR DESCRIPTION
**Pull request by dileepadev-old**
Directory for the NIBM FOSS Community @nibmfossc.

- Dileepa Bandara @dileepabandara  
  contact.dileepabandara@gmail.com  
  Club Lead, NIBM FOSS Community @nibmfossc

### CODE REVIEW

File: cc/NIBM/2022-current.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+The NIBM FOSS Community is open to any student at NIBM who are currently following any course or degree program and are interested in computing and IT. To become a member in the NIBM FOSS Community, students are required to get a free membership in the NIBM Computing Society.
+
+- Fill out the membership application form using the NIBM Microsoft 365 account.
+- Membership application form: [https://membership.nibmcs.org](https://membership.nibmcs.org)
+- Each application will be reviewed by the core team.
+- The executive committee will then get in touch with the applicant and add them as a new member of the NIBM Computing Society.
+- Members who are willing to be a part of the FOSS Community will be added to the NIBM FOSS Community.
+
+Please contact the following executives for further details:
```
Comment (dhanushkac) [inline cc/NIBM/2022-current.md]: Below part is not relavant.

File: cc/NIBM/2022-current.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  thisaruwanichamodi@gmail.com  
+  Core Team Member, NIBM FOSS Community  
+  Executive - Membership, NIBM Computing Society  
+
+## MIC/Board information
+
+- Master in Charge  
+  Mr. Thisara Weerasinghe  
+  thisara@nibm.lk  
+  [LinkedIn](https://lk.linkedin.com/in/thisara-weerasinghe-687b55a3)
```
Comment (dhanushkac) [inline cc/NIBM/2022-current.md]: This is for the tracking purpose. Just the name is enough.

**Review by dhanushkac — CHANGES_REQUESTED** [review]

**Review by dhanushkac — APPROVED** [review]
LGTM

**Comment by dileepadev-old** [comment]
Thanks @sudarakas and @dhanushkac 😊🎉

**Pull request merged**